test/traffic/response-order.test.ts

```diff
@@ -33,4 +33,11 @@ it('respects the response sequence when repeatedly requesting the same endpoint'
       },
     },
   ])
+  // Any subsequent requests to the same endpoint should return a readable response.
+  const repeatedHandlerResponse = await handlers[1]!.run({
```

Comment:
Let's put this in a separate test!

Reply:
Done, created new separate test, `response-cloning.test.ts`, with it's own archive and request script (duplicates of `request-timing`).